db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'random_data'
+require 'faker'
+
+10.times do
+    Resource.create!(
```

Comment:
you should use your factories to create the resource instead of doing it manually